docs/_docs/permalinks.md

```diff
@@ -89,7 +117,7 @@ permalink is defined according to the format `/:categories/:year/:month/:day/:ti
       </td>
       <td>
         <p>
-          Hour of the day, 24-hour clock, zero-padded from the post’s <code>date</code> front matter. (00..23)
+          Hour of the day, 24-hour clock, zero-padded from the post's <code>date</code> front matter. (00..23)
```

Comment:
i don't know how the apostrophe became curly.

Re capitalizing "post," I don't think we need to do this.